File: Solution_of_PS-Sprint-1/_02-Checking for Prime Numbers/ans.js
let userInput = prompt("Please enter a number:");
userInput = Number(userInput);

let prime = true;

if (userInput < 2) {
	prime = false;
}
for (let i = 2; i < userInput; i++) {
	if (userInput % i === 0) {
		prime = false;
        break;
	}
}
if (prime) {
	console.log(userInput + " is a prime number.");
} else {
	console.log(userInput + " is not a prime number.");
}

File: Solution_of_PS-Sprint-1/_02-Checking for Prime Numbers/ans.c
#include <stdio.h>
int main(){
    int num , isPrime = 1;
    printf("please enter a number:- ");
    scanf("%d" , &num);
    if(num<0){
        return 1;
    }
    for (int i = 2; i < num; i++)
    {
       if (num%i == 0)
       {
        isPrime = 0;
        break;
       }
       
    }
    if (isPrime)
    {
    printf("%d is a prime number" , num);
    }
    else
    {
    printf("%d is not prime number" , num);
    }
    
    
    return 0;
}